### PR TITLE
[now-cli] Rename bin to support `vercel` and `vc`

### PR DESCRIPTION
**Pull request by styfle**
We want to make sure the bin matches the [installed package name](https://docs.npmjs.com/files/package.json#bin).

This means `npm i -g now` will remain `now` and `npm i -g vercel` will use `vercel` as the binary name.

This allows support for different versions on one machine such as `npm i -g now@17 vercel@19` for example.

In addition, we will also install a shorthand `vc` so you can do `vc env pull` for example.

### CODE REVIEW

File: utils/update-legacy-name.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ if (pkg.name === '@vercel/client') {
   pkg.name = 'now-client';
 } else {
   pkg.name = pkg.name.replace('vercel', 'now');
+  if (pkg.bin && pkg.bin.vercel) {
```
Comment (TooTallNate) [inline utils/update-legacy-name.js:39]: Let's add a comment:

```suggestion

  // Rename the "vercel" binary to "now" for Now CLI
  if (pkg.bin && pkg.bin.vercel) {
```

**Review by TooTallNate — APPROVED** [review]
LGTM

File: packages/now-cli/package.json [hunk 2]
```diff
@@ -41,7 +41,6 @@
     "all": true
   },
   "bin": {
```
Comment (TooTallNate) [inline packages/now-cli/package.json:43]: Actually, we can specify the `"bin"` as a string and then it uses the field from `"name"` as the bin name. That way, the `utils/update-legacy-name.js` file doesn't need to be changed:

```suggestion
  "bin": "./dist/index.js"
```

Comment (styfle) [reply packages/now-cli/package.json:43]: Great idea! Fixed it 👍

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**